neutrons/data_workflow
# Reorganize dependencies and docker compose

Re-organize how dependencies are defined. There will be an `environment.yml` for install dependencies, and a `development.yml` which adds the dev dependencies (e.g. flake8, pytest)

Move work out of the docker-compose and remove dependencies between targets in the `Makefile`.

**To test**
```sh
docker compose down
docker image remove --force monitor-data-workflow_dasmon monitor-data-workflow_webmon
docker container prune && docker image prune && docker volume prune && docker compose up  --build
```
and click all sorts of things. The reason for removing so many things is to ensure that it is a newly built image that is used rather than one with old information.
